src/constants/property-metadata.constants.ts

```diff
@@ -431,6 +431,1468 @@ export const propertyMetadata = {
+      example: "VM300-9004-1420D935886D4B27B49E78C20FA57B93",
+    },
+    sorbentTrapSn: {
+      fieldLabels: {
+        label: "Sorbent Trap Serial Number",
+        value: "sorbentTrapSN",
+      },
+      description:
+        "Serial number (ID) for the sorbent trap on the component sampling train.  Reported as SerialNumber in Sampling Train Data record.",
+      example: "OL564824",
+    },
+    btTrapHg: {
+      fieldLabels: {
+        label: "Breakthrough Trap Hg",
+        value: "btTrapHg",
+      },
+      description:
+        "Percent breakthrough for the component sampling train sorbent trap.  Reported as PercentBreakthrough in Sampling Train Data record.",
+      example: "4.00E-04",
+    },
+    hgConcentration: {
+      fieldLabels: {
+        label: "Hg Concentration",
+        value: "hgConcentration",
+      },
+      description:
+        "Hg concentration determined from the total Hg catch and the total volume of dry gas metered for component sampling train.  Reported as HgConcentration in Sampling Train Data record.",
+      example: "2.16E-03",
+    },
+    mainTrapHg: {
+      fieldLabels: {
+        label: "Main Trap Hg",
+        value: "mainTrapHg",
+      },
+      description:
+        "Hg recovered from the main collection section (Section 1) of the sorbent trap for the component sampling train in micrograms.  Reported as MainTrapHg in SamplingTrainData record.",
+      example: "2.00E-03",
+    },
+    percentBreakthrough: {
+      fieldLabels: {
+        label: "Percent Breakthrough",
+        value: "percentBreakthrough",
+      },
+      description:
+        "Percent breakthrough for the component sampling train sorbent trap.  Reported as PercentBreakthrough in Sampling Train Data record.",
+      example: "20",
+    },
+    percentSpikeRecovery: {
+      fieldLabels: {
+        label: "Percent Spike Recovery",
+        value: "percentSpikeRecovery",
+      },
+      description:
+        "Percent spike recovery result (%) for the component sampling train sorbent trap.  Reported as PercentSpikeRecovery in Sampling Train Data record.",
+      example: "97",
+    },
+    postLeakCheckResultCode: {
+      fieldLabels: {
+        label: "Post Leak Check Result",
+        value: "postLeakCheckResultCode",
+      },
+      description:
+        "Reports whether component sampling train passed post test leak check: PASSED or FAILED.  Reported as PostLeakCheckResultCode in Sampling Train Data record.",
+      example: "PASSED",
+    },
+    referenceSfsrRatio: {
+      fieldLabels: {
+        label: "Reference SFSR Ratio",
+        value: "referenceSfsrRatio",
+      },
+      description:
+        "Reference ratio of the unadjusted hourly stack gas volumetric flow rate to the average sample flow rate for the component sampling train.  Reported as ReferenceSFSRRatio in Sampling Train Data record.",
+      example: "14.3",
+    },
+    sampleDamageExplantion: {
```

Comment:
key and fieldLabel value are both misspelled here and in the master list. Should be "sampleDamageExplanation"